cmd/gonta/main.go

```diff
@@ -23,8 +31,17 @@ func main() {
 		os.Exit(1)
 	}
 
-	d := event.NewDispatcher(logger, conf)
-	srv := server.NewGonta(logger, d, conf)
+	q := queue.New(QueueSize, logger)
+	configureCallbacks(q, conf, logger)
+
+	go q.Start()
+
+	defer func() {
+		q.Stop()
+		q.WaitUntilFinish()
+	}()
```

Comment:
This code will not work properly because Cloud Run will send a `SIGKILL` signal to the working containers to stop them.
But, any container can't handle the `SIGKILL` signal... 😑